docs/sources/trackers/mobile-trackers/tracking-events/session-tracking/index.md

```diff
@@ -212,30 +212,23 @@ Snowplow.createTracker(getApplicationContext(), namespace, networkConfig, sessio
 
 ## Decorating outgoing links using cross-navigation tracking
 
-:::note Not available before v6
+:::note
 This feature was introduced in version 6.0.0 of the iOS and Android trackers.
 :::
 
-The tracker provides a `decorateLink` API to decorate outgoing links from the mobile app to another mobile app or to a website.
-This API adds an `_sp` parameter to the links containing information about the user, app, and current session.
-This is useful for tracking the movement of users across different apps and platforms.
-It is part of our cross-navigation solution and is equivalent to [cross-domain tracking on the JavaScript tracker](/docs/sources/trackers/web-trackers/cross-domain-tracking/index.md).
-
-For example, calling `decorateLink` on `appSchema://path/to/page` will produce the following result:
+The `decorateLink` function is part of Snowplow's [cross-navigation solution](/docs/events/cross-navigation/index.md) for tracking the movement of users across different apps and platforms.
 
-```
-appSchema://path/to/page?_sp=domainUserId.timestamp.sessionId.subjectUserId.sourceId.platform.reason
-```
+Choose which parameters to include using a `CrossDeviceParameterConfiguration` object. The `domainUserId` and `timestamp` are always included automatically.
```

Comment:
Should this link to the API docs or something? The table below implies to me that we need to pass the values from the Controller/Configurations, but instead we actually just need to pass booleans (or a string for the `reason`) for each field and the tracker collects the values. This is only apparent from the examples further down ATM.